core/sql/executor/ex_hash_grby.h

```diff
@@ -302,6 +312,7 @@ NA_EIDPROC
   void resetClusterAndReadFromChild(); // Tmobile.
 
 public:
+  static const char *HashGrbyPhaseStr[];
```

Comment:
Minor comment: One way to avoid the static, global variable would be to use a method that returns a constant rather than this array.